### PR TITLE
Bug/ui

### PR DESCRIPTION
**Pull request by aniketsingh03**
This pull request fixes minor bugs pertaining to action bar in all activities. The latest apk of the app can be found at https://drive.google.com/file/d/0B6ReP5aw7ozSNVU2eXRJblhjTWM/view?usp=sharing 

Fixes:
#47 Image size is fixed on saving except for navigation bar width

### CODE REVIEW

File: app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java [hunk 1]
```diff
@@ -6,49 +6,42 @@
...
 import android.view.Window;
 import android.view.WindowManager;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 public class AboutActivity extends AppCompatActivity {
-    ImageView githubLinkBtn, reviewLinkBtn, shareLink, backBtn;
-    TextView fragTitle, openSourceLicense, versiontTextView;
-    View bottomMarginLayout;
+    ImageView githubLinkBtn, reviewLinkBtn, shareLink;
```
Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Each variable declaration must be in its own statement\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Variable 'githubLinkBtn' must be private and have accessor methods\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Variable 'reviewLinkBtn' must be private and have accessor methods\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Variable 'shareLink' must be private and have accessor methods\.

File: app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java [hunk 2]
```diff
@@ -6,49 +6,42 @@
...
+    TextView openSourceLicense, versiontTextView;
```
Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Each variable declaration must be in its own statement\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Variable 'openSourceLicense' must be private and have accessor methods\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: Variable 'versiontTextView' must be private and have accessor methods\.

File: app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java [hunk 3]
```diff
@@ -6,49 +6,42 @@
...
     PackageInfo pInfo;
     String versionName;
     int versionCode;
 
     @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if(item.getItemId()==android.R.id.home)
```
Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: 'if' construct must use '\{\}'s\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: 'if' is not followed by whitespace\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: '==' is not preceded with whitespace\.

Comment (sputnikci) [inline app/src/main/java/com/sdsmdg/kd/trianglifyexample/AboutActivity.java]: \[Checkstyle\] ERROR: '==' is not followed by whitespace\.

**Review by suyashmahar — APPROVED** [review]
Great work! Just fix those sputnik comments and we'll be ready to merge.

**Pull request merged**